config/environments/development.rb

```diff
@@ -55,7 +55,7 @@
   # Debug mode disables concatenation and preprocessing of assets.
   # This option may cause significant delays in view rendering with a large
   # number of complex assets.
-  config.assets.debug = true
+  config.assets.debug = false
```

Comment:
These settings are to override an `ActionView::Template::Error (The asset "http:/assets/application_mailer.self-b22e42926eeed27303edf398a1ca57accaf6a551e82984b53573f67b71c73a94.css" is not present in the asset pipeline` error - feel free to set back